diff --git a/node_modules/vuesax/dist/vuesax.js b/node_modules/vuesax/dist/vuesax.js
index d8f90ea..d1e5f99 100644
--- a/node_modules/vuesax/dist/vuesax.js
+++ b/node_modules/vuesax/dist/vuesax.js
@@ -18274,7 +18274,7 @@ var util_setColor = function setColor(colorName, color, el, addClass) {
 
 var insertBody = function insertBody(element, parent) {
   var target = parent ? parent : document.body;
-  target.insertBefore(element, target.lastChild);
+  target.append(element);
 };
 
 var removeBody = function removeBody(element, parent) {
